src/components/UpdateJournalEntryModal/UpdateJournalEntryModal.tsx

```diff
@@ -0,0 +1,247 @@
+import React, { useState, FormEvent, useEffect } from 'react';
+import Typography from '@mui/material/Typography';
+import FormLabel from '@mui/material/FormLabel';
+import TextField from '@mui/material/TextField';
+import DateAdapter from '@mui/lab/AdapterMoment';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import DatePicker from '@mui/lab/DatePicker';
+import Radio from '@mui/material/Radio';
+import RadioGroup from '@mui/material/RadioGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+// import { useNavigate } from 'react-router-dom';
+// import { PossibleRoutes } from '../../utils/constants';
+// import { useAuthorJournalEntry } from '../../api/journalEntries/journalEntries';
```

Comment:
No longer needed at this moment, can remove from this file but maybe save the query for possible future use